release-doc-in-process/all_vf.TTL

```diff
@@ -178,6 +182,18 @@ vf:committedOutputs
         rdfs:domain         vf:Process ;
         rdfs:range          vf:Commitment .
 
+vf:plannedFor
+        a                   owl:ObjectProperty ;
+        rdfs:label         "planned for" ;
+        rdfs:range          vf:Plan ;
+        rdfs:domain         [ owl:unionOf (vf:Process vf:Transfer) ] .
```

Comment:
I don't think `vf:Transfer` should reference a *Plan*. I think enough for Commitments to reference it.
BTW do we even have instance of `vf:Transfer` in planning phase? I would think that transfer only gets created for actual events to reference it, in planning phase we probably only have an *Agreement* referenced by Commitments